src/js/player.js

```diff
@@ -4295,7 +4299,19 @@ class Player extends Component {
       return this.language_;
     }
 
-    this.language_ = String(code).toLowerCase();
```

Comment:
we can always set this unconditionally, since it's the same line in both clauses of the if/else.
However, I think that instead of settng a property, we can add an optional argument to the language method `skipListeners` (like in the dimension method in component https://github.com/videojs/video.js/blob/b6c356d2942b3025b4acf0be0418738da9731a1b/src/js/component.js#L998). Then, in the constructor (line 405) we can call it as `this.language(this.options_.language, true)` and just have an `if (skipListeners)` around the trigger.